shard.yml

```diff
@@ -81,4 +81,4 @@ dependencies:
 development_dependencies:
   ameba:
     github: crystal-ameba/ameba
-    version: ~> 1.3.0
+    branch: master
```

Comment:
At this point, probably be better to upgrade to https://github.com/crystal-ameba/ameba/releases/tag/v1.5.0 instead of master.